content/src/content/jcr_root/apps/core/wcm/components/carousel/v2/carousel/README.md

```diff
@@ -0,0 +1,127 @@
+<!--
+Copyright 2018 Adobe Systems Incorporated
```

Comment:
• There is a new copyright notice since this was implemented (basically "Adobe Systems Incorporated" -> "Adobe")
• Should make sure the year is 2019
• Should apply the same to each file in the PR

See:

```
<!--
Copyright 2019 Adobe

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->
```